packages/datatrak-web/src/components/ChangeProjectButton.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Box } from '@material-ui/core';
+import { useCurrentUser } from '../api';
+import { ProjectSelectModal } from '../layout/UserMenu/ProjectSelectModal.tsx';
+import { Button, TooltipButtonWrapper } from './Button.tsx';
+
+/**
+ * Ensures inline placement when appropriate. Necessary because tooltip attribute on the button
+ * wraps it in an element whose outer `display` type is `block`.
+ */
+const Container = styled(Box)`
+  .MuiTypography-h1 &,
+  .MuiTypography-h2 &,
+  .MuiTypography-h3 &,
+  .MuiTypography-h4 &,
+  .MuiTypography-h5 &,
+  .MuiTypography-h6 & {
```

Comment:
I wonder if you can just use something like `MuiTypography-root` or some equivalent instead of having to explicitly declare all of these? Or alternatively, you can give a prop of `display='inline'` to any offending typography elements, as I think it unlikely we will have whatever issue is a problem here on all 6 heading levels